skore/src/skore/item/cross_validation_item.py

```diff
@@ -357,23 +288,14 @@ def factory(
             "hash": _hash_numpy(X_array),
         }
 
-        # Keep plot itself as well as bytes so we can cache it
-        plot = plot_cross_validation(cv_results_serialized)
-        plot_bytes = plotly.io.to_json(plot, engine="json").encode("utf-8")
```

Comment:
Why delegate the transformation to the caller?
I think it was the right place to call `plot_bytes = plotly.io.to_json(plot, engine="json").encode("utf-8")`?

```
    @classmethod
    def factory_raw(
        cls,
        cv_results: dict,
        estimator: sklearn.base.BaseEstimator,
        X: Data,
        y: Target | None,
        plot: [...],
    ) 
```

That case, your function signature is better coherent: it only accepts "raw" objects.

Reply:
you're right, it's more coherent this way. I'll add move it back

Reply:
done